scala/scala.bzl

```diff
@@ -446,39 +464,42 @@ def _scala_binary_impl(ctx):
   cjars += [ctx.file._scalareflect]
   rjars += [ctx.outputs.jar, ctx.file._scalalib, ctx.file._scalareflect]
   rjars += _collect_jars(ctx.attr.runtime_deps).runtime
-  _write_launcher(ctx, rjars)
+  _write_launcher(
+      ctx = ctx,
+      rjars = rjars,
+      main_class = ctx.attr.main_class,
+      jvm_flags = "",
```

Comment:
seems like we should want to support jvm_flags on scala_binary and now it is easy. Can we do that? They should be in the common attributes: `" ".join(ctx.attr.jvm_flags)` I think.